middleware/auth.js

```diff
@@ -108,15 +109,11 @@ module.exports = function (params) {
           // TODO:?
           req.openidState = decodeState(expectedState);
 
-          // intentional clone of the properties on tokenSet
-          Object.assign(req[config.session.name], {
-            id_token: tokenSet.id_token,
-            access_token: tokenSet.access_token,
-            refresh_token: tokenSet.refresh_token,
-            token_type: tokenSet.token_type,
-            expires_at: tokenSet.expires_at,
-          });
+          if (config.afterCallback) {
+            session = await config.afterCallback(req, res, session); 
```

Comment:
You ned to pass in the `decodedState` (or change the ts defs) - although I'm inclined to suggest passing it in since we'll need to do this in the next sdk https://github.com/auth0/nextjs-auth0/blob/master/src/handlers/callback.ts#L57